red/Makefile

```diff
@@ -0,0 +1,11 @@
+STEPS = step0_rep.redl step1_read_print.red step2_eval.red step3_env.red step4_if_fn_do.red \
+		step5_tco.red step6_file.red step7_quote.red step8_macros.red step9_try.red stepA_mal.red
+
+all:
+	$(patsubst %.red,%,$(STEPS))
+
+step%: step%.red
+	red-064 -c $<
+
+clean:
+	rm -f $(STEPS:%.red=%) *.red mal
```

Comment:
clean removes `*.red`? that sounds bad.

Reply:
I wasn't quite sure how this was supposed to work, so I used the one from pascal as reference

Reply:
@ALANVF `*.red` there means clean will remove all your source files. The intention of clean is to remove all generated files, not source files.

Reply:
Ah that makes more sense. I saw it in the pascal makefile so I assumed that I needed it.